cocos2d/core/sprites/CCScale9Sprite.js

```diff
@@ -891,7 +864,12 @@ cc.Scale9Sprite = _ccsg.Node.extend({
         this._blendFunc = cc.BlendFunc._alphaNonPremultiplied();
         this._fillCenter = cc.v2(0,0);
         this.setAnchorPoint(cc.p(0.5, 0.5));
-        //
+        // Init vertex data for simple
+        this._rawVerts = null;
+        this._rawUvs = null;
+        this._vertices = dataPool.get(8) || new Float32Array(8);
+        this._uvs = dataPool.get(8) || new Float32Array(8);
```

Comment:
Use total flat data structure, and remove webgl buffer, because we will upload data directly to the global buffer for auto batching